src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,166 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> currentNode;
+    private Node<T> previousNode;
+    private Node<T> head;
+    private Node<T> tail;
+
+    static class Node<T> {
```

Comment:
```suggestion
    private static class Node<T> {
```
And move it to the end of the class